docs/dx_tables.md

```diff
@@ -1,549 +1,74 @@
-normally direct constituents of a particular Table A mnemonic. In other words, at the highest level,
+## Mnemonics
+
+A <i>mnemonic</i> is a
+descriptive, alphanumeric name for a data value.
+* "Table A mnemonics", refer to particular data subset (i.e. report ) types,
+* "Table B mnemonics", refer directly to basic data values,
+* "Table D mnemonics" are
+sequences composed of one or more Table B (or other Table D) mnemonics and which are themselves
+normally direct constituents of a particular Table A mnemonic.
+
+<br>
+
+At the highest level,
 we have a Table A mnemonic which completely describes a type of data subset (e.g. rawinsonde,
-wind profiler, etc.), and this Table A mnemonic is defined as a sequence of one or more Table B or
+wind profiler, etc.). This Table A mnemonic is defined as a sequence of one or more Table B or
 Table D mnemonics, where each Table D mnemonic is likewise itself defined as a sequence of one or
 more Table B or Table D mnemonics, and so on until the entire data subset can be equivalently
-described as a sequence of one or more Table B mnemonics which, again, themselves correspond to
-basic data values (e.g. pressure, temperature, humidity, etc.). In this way, the entire sequence
-of data values that constitute a particular type of data subset is fully and unambiguously defined,
-both for purposes of input (reading/decoding) or output (writing/encoding) of reports corresponding
-to that particular type of data subset.
+described as a sequence of one or more Table B mnemonics which correspond to
+basic data values (e.g. pressure, temperature, humidity, etc.).
 
 <br>
 
-However, it's also important to understand what mnemonics are not. Specifically, mnemonics never
-themselves appear within actual BUFR messages that are read or written by the BUFRLIB software;
-rather, their only purpose in life is to make it easier for users to interact with the software by
-providing descriptive names to represent individual data values, as opposed to having to keep track
-of the corresponding FXY numbers (described below), which are much less intuitive but which
-nevertheless are the prescribed method within the BUFR code form for referencing of individual data
-values, and which therefore are what are actually read and written by the software.
+The entire sequence
+of data values that constitute a particular type of data subset is fully and unambiguously defined.
 
 <br>
 
-Before continuing on let's recall
-that a DX BUFR tables file consists of three distinct sections.  Each section contains one or more
+Mnemonics never
+themselves appear within actual BUFR messages.
+Their only purpose is to make it easier for users to interact with the software by
+providing descriptive names to represent individual data values. They are a more intuitive than
+FXY numbers (described below). FXY numers are the prescribed method within the BUFR messages.
+
+## DX BUFR Tables File
```

Comment:
How about changing the last 2 sentences to "They are more intuitive than
FXY numbers (described below), which are the prescribed method within actual BUFR messages."?

Otherwise, if you think that's too wordy or unclear, then please at least remove the superfluous "a" near the start of the 1st sentence, and correct the spelling of numers to numbers in the 2nd sentence.

Reply:
done